app/src/main/java/com/sdp/movemeet/view/navigation/Navigation.java

```diff
@@ -185,25 +189,26 @@ public void onComplete(@NonNull Task<DocumentSnapshot> task) {
     public void createDrawer() {
         drawerLayout = activity.findViewById(R.id.drawer_layout);
         navigationView = activity.findViewById(R.id.nav_view);
-        TextView textView = activity.findViewById(R.id.textView);
+        //TextView textView = activity.findViewById(R.id.textView);
         Toolbar toolbar = activity.findViewById(R.id.toolbar);
 
         navigationView.bringToFront();
         ActionBarDrawerToggle toggle = new
                 ActionBarDrawerToggle(activity, drawerLayout, toolbar, R.string.navigation_drawer_open, R.string.navigation_drawer_close);
         drawerLayout.addDrawerListener(toggle);
 
-        View hView = navigationView.inflateHeaderView(R.layout.header);
+        //View hView = navigationView.inflateHeaderView(R.layout.header);
+        navigationView.inflateHeaderView(R.layout.header);
 
-        TextView fullName = hView.findViewById(R.id.text_view_profile_name);
+        /*TextView fullName = hView.findViewById(R.id.text_view_profile_name);
         TextView phone = hView.findViewById(R.id.text_view_profile_phone);
-        TextView email = hView.findViewById(R.id.text_view_profile_email);
+        TextView email = hView.findViewById(R.id.text_view_profile_email);*/
```

Comment:
Why don't you simply remove those lines since we plan not to print the user info in the drawer anymore?

Reply:
That's right, I will delete all my commented lines before merge!